Formula/libtensorflow.rb

```diff
@@ -13,18 +15,24 @@ class Libtensorflow < Formula
 
   depends_on "bazel" => :build
   depends_on :java => ["1.8", :build]
+  depends_on "python@2" => :build
 
-  # Allow libtensorflow to be built on bazel 0.22.0
-  patch do
-    url "https://github.com/tensorflow/tensorflow/commit/91da898cb6f6b0e751e15ceb813a37cdfe18a035.patch?full_index=1"
-    sha256 "648295170a4d4226a76f916e61bf052dcd4b13e1c0517386a0e59963285cdc9b"
-  end
+  # Allow libtensorflow to be built on bazel 0.26.1
+  # this patch a consolidation of the following patches
+  # - https://github.com/tensorflow/tensorflow/commit/c33055c44226499e99270ed05265ae3b960210c9 (from 0.25.2 to 0.25.3)
+  # - https://github.com/tensorflow/tensorflow/commit/040546fdefc5a111496dcd7be97b7d65594f9119 (from 0.25.3 to 0.26.0)
+  # - https://github.com/tensorflow/tensorflow/commit/c3260eb5ab384e4d03e7e8c840712380492b26d9 (from 0.26.0 to 0.26.1)
+  patch :DATA
 
   def install
+    venv_root = "#{buildpath}/venv"
+    venv = virtualenv_create(venv_root, "python2")
+    venv.pip_install("future>=0.17.1") # mimic changes from https://github.com/tensorflow/tensorflow/pull/29233 and https://github.com/tensorflow/tensorflow/pull/29138
```

Comment:
Can we use these as a patch now that they're merged?

Reply:
I don't think so, both https://github.com/tensorflow/tensorflow/pull/29233 and https://github.com/tensorflow/tensorflow/pull/29138 changes build scripts which homebrew formula doesn't use.